### ISSUE TITLE
[Impact Trump Cards] Different card templates in the repo then on webinar

### ISSUE THREAD

**Issue by adamlukaszczyk** [impactmapping/open-impact-mapping-workshop#11]
Hi, in the cards template we can find this statement of impact for each actor:
`who ... can help/obstruct ... how` ([link](https://github.com/impactmapping/open-impact-mapping-workshop/blob/master/facilitation-games/impact-trump-cards/handouts/cards-template-color.jpg))

And in the talk available on Youtube _Gojko Adzic: Impact Mapping with Innovation Games_ template statement is:
`who ... will do ... instead off ...` ([link](https://youtu.be/aI5hJ6IMC9w?t=2529))

From where is that difference? Which one is better to use?

**Comment by gojko** [comment]
the cards are there to spark a discussion, and using different cards even in the same conversation can lead to more ideas - so both are good. having said that, I prefer to describe a change in behaviour, so doing something instead of something else is slightly more on point

**Comment by adamlukaszczyk** [comment]
Yes, I will try that! I will mix the cards to foster creativity. Thanks
